File: spring-data-requery-java/src/main/java/com/coupang/springframework/data/requery/repository/query/RequeryParameterAccessor.java
package com.coupang.springframework.data.requery.repository.query;

import org.springframework.data.repository.query.ParameterAccessor;

/**
 * RequeryParameterAccessor
 *
 * @author debop@coupang.com
 * @since 18. 6. 8
 */
public interface RequeryParameterAccessor extends ParameterAccessor {

    RequeryParameters getParameters();

}

File: src/main/kotlin/com/coupang/springframework/data/requery/repository/query/RequeryParameterAccessor.kt
package com.coupang.springframework.data.requery.repository.query

import org.springframework.data.repository.query.ParameterAccessor

/**
 * RequeryParameterAccessor
 *
 * @author debop@coupang.com
 * @since 18. 5. 29
 */
interface RequeryParameterAccessor: ParameterAccessor {

    fun getParameters(): RequeryParameters

    // fun getQueryOptions(): RequeryQueryOptions

    fun getBindVars(): Map<String, Any?>
}